apps/docs/src/app/core/component-docs/inline-help/inline-help-header/inline-help-header.component.html

```diff
@@ -1,7 +1,11 @@
 <header>Inline Help</header>
 <description
     >Inline help is used to display help text in a popover, often inline with headers, body text and form labels. It
-    also uses the fd-popover component and supports much of its functionality.</description
+    also uses the fd-popover component and supports much of its functionality.
+    <fd-message-strip [type]="'warning'" [dismissible]="false">
```

Comment:
unnecessary binding `[type]="'warning'"` => `type="warning"`

Reply:
fixed. Thanks for the catch